app/routes.py

```diff
@@ -0,0 +1,15 @@
+from flask import Blueprint, request, jsonify
+import app.services.message_status.main as message_status
+
+api = Blueprint("api", __name__)
+
+@api.route("/message-status/create", methods=["POST"])
```

Comment:
We should probably rename this endpoint to `/status/create` as it will handle message and channel status callbacks.

Reply:
Ah yes fair point, that was just on my TODO list but I agree we should do it after this PR is merged and we're happy with our deployed app.